spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -278,6 +278,11 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
         fun parse(expression: String): SpdxSingleLicenseExpression =
             SpdxExpression.parse(expression) as SpdxSingleLicenseExpression
     }
+
+    /**
+     * Return the string identifier of this license, without the optional exception.
+     */
+    abstract fun licenseId(): String
```

Comment:
How about calling this `simpleLicense` to resemble `SpdxSimpleExpression`?

Reply:
Done.